src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -199,57 +201,18 @@ class Unsupported(Node):
     # pylint: enable=abstract-method
 
     unsupported = Unsupported()
-    assignment = schedule.children[0]
+    assignment = schedule[0]
     binary_operation = assignment.children[1]
```

Comment:
No idea why this is in the left column now ...
Isn't this assigment.rhs?

Reply:
Changed to assignment.rhs. You can't modify rhs so I've added a comment before the reference (children[1]) is modified.